wdtk-client/src/main/java/org/wikidata/wdtk/client/RdfSerializationAction.java

```diff
@@ -251,4 +254,21 @@ private void printHelp() {
 					+ TASK_HELP.get(supportedTask), 75, "\n   ", true));
 		}
 	}
+
+	@Override
+	public String getReport() {
+		String message = "Finished serialization of "
+				+ this.serializer.getTripleCount() + " RDF triples in file "
+				+ this.outputDestination;
+		return message;
+	}
+
+	@Override
+	public String getActionName() {
```

Comment:
Could we move this to the abstract super class and use the local name of this.class() as a default? Alternatively, could we just require a default action name as a constructor parameter to the abstract super class. Then the superclass can implement getActionName() (once and for all) without having to repeat this code in each subclass. But using the local class name seems even simpler.